src/client/modules/my/dataUpload/dataUpload.ts

```diff
@@ -0,0 +1,62 @@
+import { LightningElement } from 'lwc';
+
+export default class DataUploader extends LightningElement {
+
+    /**
+     * Retrieves the data from the input fields and makes a query to upload it to the database api.
+     */
+    gatherDataAndSend() {
+
+        let playerData = []; //data for each player input
+
+        //get entered player data
+        for(let x = 0; x < 6; x++) {
+            playerData.push({
+                "playerName": this.getValueFromInput("playerName" + (x+1).toString()),
+                "victoryPoints": parseInt(this.getValueFromInput("victoryPoints" + (x+1).toString()), 10)
+            });
+        }
+
+        //data for post
+        let data = {
+            "gameId": this.getValueFromInput("gameId"),
+            "playerData": playerData
+        };
+
+        console.log(data);
+        console.log(JSON.stringify(data));
+
+        //send POST request to api
+        fetch("api/v1/gameLogs", {
```

Comment:
Where's the server code for this? Did you forget to add api.ts ?

Reply:
BethAnne is working on the server side in her repo, so the idea was to merge the client side request so she could test with it. If it would be better to do both client and server side pieces in one repo. we can do that instead.